generated-sources.json

```
@@ -29482,13 +29482,13 @@
     },
     {
         "type": "git",
-        "url": "https://git@github.com/laurent22/node-emoji.git",
+        "url": "ssh://git@github.com/laurent22/node-emoji.git",
```

```suggestion
        "url": "https://github.com/laurent22/node-emoji.git",
```

This is probably what you want.